string robot_id
geometry_msgs/PoseStamped home_pose
---
bool success
string message